From 1cb53934b83b794c319813106c9f12d75cce66d2 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Sun, 18 Jan 2026 19:24:42 -0800
Subject: [PATCH] macros.in: Set _pkgverify_level to digest as RPM 4.20.1

Fixed:
$ bitbake core-image-minimal
Package bash-5.3-r0.16.x86_64_v3.rpm is not signed
GPG check FAILED

Set _pkgverify_level to digest as RPM 4.20.1 can fix the problem.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
Signed-off-by: Sumanth Gavini <sumanth.gavini@windriver.com>
---
 macros.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/macros.in b/macros.in
index 38066f00c..7fe00ad09 100644
--- a/macros.in
+++ b/macros.in
@@ -680,7 +680,7 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 # signature	require valid signature(s)
 # digest	require valid digest(s)
 # none		traditional rpm behavior, nothing required
-%_pkgverify_level all
+%_pkgverify_level digest
 
 # Disabler flags for package verification (similar to vsflags)
 # Set to 0x0 for full compatibility with v4 packages.
-- 
2.49.0

